tests/heat_flux_map_vis_point_wise.prm

```diff
@@ -0,0 +1,106 @@
+# This is a copy of the heat_flux_map_vis postprocessor that
```

Comment:
Rather than copy most of the parameter file over, this seems like a good case to derive from the existing `heat_flux_map_vis` test.